src/amo/components/Addon/index.js

```diff
@@ -338,6 +338,29 @@ export class AddonBase extends React.Component {
     /* eslint-enable react/no-danger */
   }
 
+  renderAddonsByAuthorsCard(isForTheme) {
```

Comment:
Could we use object destructuring here?

```js
renderAddonsByAuthorsCard({ isForTheme }) { // ...
```

so that we can call it as follows:

```js
{this.renderAddonsByAuthorsCard({ isForTheme: true })}
```

Reply:
My idea is to avoid method calls using boolean arguments without context.

Reply:
Yes, good idea. 👍  Fixed.